File: AdminPanelUI/src/types/socialLink.ts
export interface SocialLink {
  id: number
  platformName: string
  url: string
  iconClass: string | null
  sortOrder: number
}

export interface SocialLinkListItem {
  id: number
  platformName: string
  url: string
}

export interface SaveSocialLinkPayload {
  platformName: string
  url: string
  iconClass: string | null
  sortOrder: number
}

File: AdminAPI/Models/SocialLink.cs
namespace AdminAPI.Models;

public class SocialLink
{
    public int Id { get; set; }

    public string PlatformName { get; set; } = string.Empty;

    public string Url { get; set; } = string.Empty;

    public string? IconClass { get; set; }

    public int SortOrder { get; set; }
}
